### ISSUE TITLE
Application not being listed and themes

### ISSUE THREAD

**Issue by gpapaiko** [owncloud/web#5116]
Hi,

Today I just got this new interface configured after I did the update to the latest version via marketplace update.
Currently on version 3.1.0
![image](https://user-images.githubusercontent.com/21032399/118594245-79e2be80-b7ec-11eb-979e-17a5a54cf047.png)

On my current (classical view I can see all of these application), I use most of them other are for testing.
![image](https://user-images.githubusercontent.com/21032399/118594320-9aab1400-b7ec-11eb-9110-c66f5efd6d83.png)

On the new WEB interface I cannot see any of them, how can I get them to show up.
After a little bit of playing around with the config.json file I mange tom get a couple to show up, but I cannot get the icon to be the correct on, how can I fix this.
![image](https://user-images.githubusercontent.com/21032399/118594364-af87a780-b7ec-11eb-96bb-8a0b17df8c1a.png)

CONFIG.JSON
```
{
  "server" : "https://cloud.xx.info",
  "theme": "ins-theme",
  "auth": {
    "clientId": "xxxxx",
    "url": "https://cloud.xx.info/index.php/apps/oauth2/api/v1/token",
    "authUrl": "https://cloud.xx.info/index.php/apps/oauth2/authorize"
  },
  "apps" : [
    "files",
    "media-viewer",
    "draw-io",
    "contacts",
    "activity",
    "calendar",
    "gallery"
  ],
  "applications" : [
    {
      "title": {
        "en": "Classic Design",
        "de": "Dateien",
        "fr": "Fichiers",
        "zh_CN": "文件"
      },
      "icon": "switch_ui",
      "url": "https://cloud.xx.info/index.php/apps/files"
    },
    {
      "title": {
        "en": "Contacts"
      },
      "icon": "/apps/contacts/img/app.svg",
      "url": "https://cloud.xx.info/index.php/apps/contacts/"
    },
    {
      "title": {
        "en": "Calendar"
       },
      "icon": "application",
      "url": "https://cloud.xx.info/index.php/apps/calendar"
    },
    {
      "icon": "application",
      "menu": "user",
      "target": "_self",
      "title": {
        "en": "Settings",
        "de": "Einstellungen"
      },
      "url": "https://cloud.xx.info/index.php/settings/personal"
    }
  ]
}
```
Do I have to manually add all the application I want to use?
If so how can I get the to use the correct icon for the application?

There should be an option to replicate the current application menu that eh user has.

This section (see below) in the config file seem not make any difference or have any affect on the interface or OC, what is its funtion?
 ```
"apps" : [
    "files",
    "media-viewer",
    "draw-io",
    "contacts",
    "activity",
    "calendar",
    "gallery"
  ],
```

Also if I click on one of the application I had link to et contacts, the screen (page layout) still reverts back to the orignal classical view.

Is there any plans to change that?

Also in relation to THEMES, I have kind of figured out were the themes of this new WEB interface are, but if I add my own to that location (see below) will that generate any errors or application integrity checks? 
the themes in this location also seems to be limited to just changing the icon, background image and slogan, is there any plans to use the current themes model?

I have tried to use my custom theme but that did not work.

```
root@inscloud:**/var/www/owncloud/apps/web/themes**# ll
drwxr-xr-x 3 www-data www-data 4096 May 18 13:05 owncloud/
root@inscloud:/var/www/owncloud/apps/web/themes#
```

Thanks in advance.

**Comment by pascalwengerter** [comment]
@gpapaiko will get back to you about the app situation at a later point, regarding the theming the docs are to be found here: https://owncloud.dev/clients/web/theming/

TL;DR: Place a `theme.json` (find template at the bottom of the docs page)  in web/packages/web-runtime/themes/{theme-name}/theme.json and change the `config.json` accordingly, or reference a theme as an URL. Need to restart the app to load the new theme (and the original OC theme will get loaded if things go wrong as a fallback)

**Comment by pascalwengerter** [comment]
Finally got around to looking this today. To summarize the questions you have:

1. Do I have to manually add all the application I want to use?
As of now, yes

2. If so how can I get the to use the correct icon for the application?
You can use any of the icons from our [design system icons](https://owncloud.design/#/Design%20Tokens/Icon) and open an issue there if you need any that are not available

3. There should be an option to replicate the current application menu that the user has.
Agreed, yet that is currently not possible

4. This section (see below) in the config file seem not make any difference or have any affect on the interface or OC, what is its function?
Those are the applications that are being shipped with `web`, by adding/omitting apps they are (not) being shipped to your users. Please see https://github.com/owncloud/web/tree/master/packages for the ones that are available (but also notice there are some you can not disable, e.g. the runtime and the oc10-integration)

5. Also if I click on one of the application I had link to et contacts, the screen (page layout) still reverts back to the original classical view. Is there any plans to change that?
This is expected behavior - you link currently running "old" applications in the `applications` array in the config, whilst new `apps` are being shipped with the new `web` project. We're working on integrations and new/better apps at the moment

6. Regarding themes
I think I answered your questions in the separate issue you opened and linked the docs, please reply to any of the two issues if more questions arise!

Please see a cleaned-up config below

```json
{
  "server": "https://cloud.xx.info",
  "theme": "https://path-to-your-theme.info/example/theme.json",
  "auth": {
    "clientId": "xxxxx",
    "url": "https://cloud.xx.info/index.php/apps/oauth2/api/v1/token",
    "authUrl": "https://cloud.xx.info/index.php/apps/oauth2/authorize"
  },
  "apps": [
    "files",
    "media-viewer",
    "draw-io"
  ],
  "applications": [
    {
      "title": {
        "en": "Classic Design",
        "de": "Dateien",
        "fr": "Fichiers",
        "zh_CN": "文件"
      },
      "icon": "switch_ui",
      "url": "https://cloud.xx.info/index.php/apps/files"
    },
    {
      "title": {
        "en": "Contacts"
      },
      "icon": "group",
      "url": "https://cloud.xx.info/index.php/apps/contacts"
    },
    {
      "title": {
        "en": "Calendar"
      },
      "icon": "text-calendar",
      "url": "https://cloud.xx.info/index.php/apps/calendar"
    },
    {
      "icon": "application",
      "menu": "user",
      "target": "_self",
      "title": {
        "en": "Settings",
        "de": "Einstellungen"
      },
      "url": "https://cloud.xx.info/index.php/settings/personal"
    }
  ]
}
```
